File: java/Baju.java
class Baju extends Aksesoris
{
    //atribut
    private String size;
    private String merk;

    //constructor
    public Baju()
    {
        this.size = "";
        this.merk = "";
    }
    public Baju(String size, String merk)
    {
        this.size = size;
        this.merk = merk;
    }

    //setter
    public void setsize(String size)
    {
        this.size = size;
    }
    public void setmerk (String merk)
    {
        this.merk = merk;
    }

    //getter
    public String getsize()
    {
        return this.size;
    }
    public String getmerk()
    {
        return this.merk;
    }
}

File: php/Baju.php
<?php
//library
require('Aksesoris.php');

//deklarasi class Baju yang menjadi anak class Aksesoris
class Baju extends Aksesoris
{
     //private atribut
     private $size;
     private $merk;
     
     
     //constructor
     public function __construct($size = '', $merk = '')
     {
         $this->size = $size;
         $this->merk = $merk;
     }
 
     //setter
     public function setsize($size)
     {
         $this->size = $size;
     }
     public function setmerk($merk)
     {
         $this->merk = $merk;
     }
     
     
     //getter
     public function getsize()
     {
         return $this->size;
     }
     public function getmerk()
     {
         return $this->merk;
     }
     
     
}
?>
